src/main/io/vtx_smartaudio.c

```diff
@@ -497,6 +497,7 @@ static void saReceiveFrame(uint8_t c)
 static void saSendFrame(uint8_t *buf, int len)
 {
     if (!IS_RC_MODE_ACTIVE(BOXVTXCONTROLDISABLE)) {
+        #ifndef AT32F4
```

Comment:
This should probably apply to most current targets ...

Reply:
so the bidir first byte skip bug is fixed for other targets too?
Current implementation works on F7 for example (with the 0x00 byte)